### ISSUE TITLE
DEVELOPMENT HAS BEEN MOVED | New maintainer of the html2text project

### ISSUE THREAD

**Issue by Alir3z4** [aaronsw/html2text#91]
Hey,

tl;dr: **https://github.com/Alir3z4/html2text**

It's been almost 3 years since the the last release, version `3.02`. As myself, I use this package on some projects and it works perfectly.

Recently I found out the author is not alive anymore(sadly), and there're couple of really helpful pull-request waiting for review and get merged into master. Although we can fork the repo and use it on our own but the original package on the pypi would remains untouched and eventually will be outdated.

Currently I'm the maintainer of `html2text` on the pypi:
https://pypi.python.org/pypi/html2text

I bumped the version number to `2014.4.5` and did a release. No changes has been made on the original code(mainly to not introduce conflicts with current pull-requests and unmerged patches), I only did:
- Add `ChangeLog.rst` file.
- Add `AUTHORS.rst` file.
- Update `README.md`.

Also I merged some changes from @mcepl fork because they were look good. It would be great to see others send their patches to my repo where I'll keep the latest changes:

https://github.com/Alir3z4/html2text

I'll try to do code review as much as I can, but I hope to get some help from @mcepl and others who are already familiar with the code base and its concept.

Thanks,
Alireza Savand

**Comment by mcepl** [comment]
> Currently I'm the maintainer of html2text on the pypi:
> https://pypi.python.org/pypi/html2text

How did you manage to take over Aaron’s PyPI account?

**Comment by Alir3z4** [comment]
@mcepl I didn't take over anybody's account, I'm just the maintainer of it. @aaronsw is still the owner of the account and the package.

**Comment by mcepl** [comment]
Do you know by chance whether there is any progress in resolving his inheritance? (http://www.aaronsw.com/2002/continuity)

**Comment by Alir3z4** [comment]
Huh, I didn't know about something like that. I just read it.
Might be a good idea to contact Sean?

**Comment by mcepl** [comment]
You’ve got mail (I hope).

**Comment by mcepl** [comment]
I think the best solution is to have https://github.com/html2text/html2text and work on working on pull reuqests (and tickets) there. What do you think?

**Comment by jonathan-s** [comment]
The maintained version can be found here: https://github.com/Alir3z4/html2text/

**Comment by arpalavi** [comment]
thank you for the new location update
